src/services/itemLogin/index.ts

```diff
@@ -34,9 +39,34 @@ const plugin: FastifyPluginAsync = async (fastify) => {
     '/:id/login-schema-type',
     { schema: getLoginSchemaType, preHandler: optionalIsAuthenticated },
     async ({ user, params: { id: itemId } }) => {
-      const value =
-        (await itemLoginService.getSchemaType(user?.account, buildRepositories(), itemId)) ?? null;
-      return value;
+      return await db.transaction(async (manager) => {
+        const repositories = buildRepositories(manager);
+        const item = await itemService.get(
+          user?.account,
+          repositories,
+          itemId,
+          PermissionLevel.Read,
+          false,
+        );
+
+        const tags = await itemTagService.getByItemPath(repositories, item.path);
+        const isHidden = Boolean(tags.find((t) => t.type === ItemTagType.Hidden));
+        if (
+          isHidden &&
+          (user?.account == undefined ||
```

Comment:
```suggestion
          (!user?.account ||
```

Reply:
Since `user` has type `any`, there's a way that `account` is a boolean. (Which should never happen...). So I prefer to be explicit here. Otherwise I can move this check in an utils function

Reply:
Okay so in this case
```suggestion
          (user?.account === undefined ||
```